### PR TITLE
Added Section - Inactive Mailboxes and Auto-Expanding Archive

### PR DESCRIPTION
**Pull request by Tangela123**
Recover and Restore cmdlets for Inactive Mailboxes are not supported for mailboxes configured for Auto-Expanding Archive mailbox.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/inactive-mailboxes-in-office-365.md [hunk 1]
```diff
@@ -65,7 +65,14 @@ You might consider creating an Office 365 retention policy specifically for inac
...
   
 For more information about eDiscovery cases and holds, see [eDiscovery cases](ediscovery-cases.md).
-  
+
+## Inactive Mailboxes and Auto-Expanding Archive
+ An inactive mailbox that is configured with an Auto-Expanding Archive cannot be recovered or restored. In scenarios, where it is necessary to recover data for these specific types of mailboxes it is recommended that the data is exported from the mailbox using the Content Search tool and imported to the desired location. For step-by-step procedures for searching mailboxes and exporting search results, see the following topics:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/inactive-mailboxes-in-office-365.md]:
```suggestion
 An inactive mailbox that is configured with an Auto-Expanding Archive cannot be recovered or restored. In scenarios where it is necessary to recover data for these specific types of mailboxes, it is recommended that the data is exported from the mailbox using the Content Search tool and imported to the desired location. For step-by-step procedures for searching mailboxes and exporting search results, see the following topics:
```

File: microsoft-365/compliance/inactive-mailboxes-in-office-365.md [hunk 2]
```diff
@@ -65,7 +65,14 @@ You might consider creating an Office 365 retention policy specifically for inac
...
   
 For more information about eDiscovery cases and holds, see [eDiscovery cases](ediscovery-cases.md).
-  
+
+## Inactive Mailboxes and Auto-Expanding Archive
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/inactive-mailboxes-in-office-365.md]:
```suggestion
## Inactive mailboxes and auto-expanding archive
```

File: microsoft-365/compliance/inactive-mailboxes-in-office-365.md [hunk 3]
```diff
@@ -65,7 +65,14 @@ You might consider creating an Office 365 retention policy specifically for inac
...
+ An inactive mailbox that is configured with an Auto-Expanding Archive cannot be recovered or restored. In scenarios, where it is necessary to recover data for these specific types of mailboxes it is recommended that the data is exported from the mailbox using the Content Search tool and imported to the desired location. For step-by-step procedures for searching mailboxes and exporting search results, see the following topics:
+
+Content Search in Office 365
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/inactive-mailboxes-in-office-365.md]:
```suggestion
[Content Search in Office 365](https://docs.microsoft.com/microsoft-365/compliance/content-search?view=o365-worldwide)
[Export Content Search results](https://docs.microsoft.com/microsoft-365/compliance/export-search-results?view=o365-worldwide)
```

File: microsoft-365/compliance/inactive-mailboxes-in-office-365.md [hunk 4]
```diff
@@ -65,7 +65,14 @@ You might consider creating an Office 365 retention policy specifically for inac
...
+Export Content Search results
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/inactive-mailboxes-in-office-365.md]:
```suggestion
```

File: microsoft-365/compliance/inactive-mailboxes-in-office-365.md (hunk 3 above)
Comment (JohanFreelancer9) [inline microsoft-365/compliance/inactive-mailboxes-in-office-365.md]:
```suggestion
```

File: microsoft-365/compliance/inactive-mailboxes-in-office-365.md [hunk 5]
```diff
@@ -65,7 +65,14 @@ You might consider creating an Office 365 retention policy specifically for inac
...
   
 For more information about eDiscovery cases and holds, see [eDiscovery cases](ediscovery-cases.md).
-  
+
+## Inactive Mailboxes and Auto-Expanding Archive
+ An inactive mailbox that is configured with an Auto-Expanding Archive cannot be recovered or restored. In scenarios, where it is necessary to recover data for these specific types of mailboxes it is recommended that the data is exported from the mailbox using the Content Search tool and imported to the desired location. For step-by-step procedures for searching mailboxes and exporting search results, see the following topics:
+
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/inactive-mailboxes-in-office-365.md]:
```suggestion
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - suggestions.

File: microsoft-365/compliance/inactive-mailboxes-in-office-365.md (hunk 2 above)
Comment (JohanFreelancer9) [inline microsoft-365/compliance/inactive-mailboxes-in-office-365.md]:
```suggestion
## Inactive mailboxes and auto-expanding archive
```

File: microsoft-365/compliance/inactive-mailboxes-in-office-365.md (hunk 1 above)
Comment (JohanFreelancer9) [inline microsoft-365/compliance/inactive-mailboxes-in-office-365.md]:
```suggestion
 An inactive mailbox that is configured with an auto-expanding archive cannot be recovered or restored. In scenarios where it is necessary to recover data for these specific types of mailboxes, it is recommended that the data is exported from the mailbox using the Content Search tool and imported to the desired location.
 For step-by-step procedures to search mailboxes and export search results, please see [Content Search in Office 365](https://docs.microsoft.com/microsoft-365/compliance/content-search?view=o365-worldwide) and [Export Content Search results](https://docs.microsoft.com/microsoft-365/compliance/export-search-results?view=o365-worldwide).
```

File: microsoft-365/compliance/inactive-mailboxes-in-office-365.md (hunk 5 above)
Comment (JohanFreelancer9) [inline microsoft-365/compliance/inactive-mailboxes-in-office-365.md]:
```suggestion
```

File: microsoft-365/compliance/inactive-mailboxes-in-office-365.md (hunk 3 above)
Comment (JohanFreelancer9) [inline microsoft-365/compliance/inactive-mailboxes-in-office-365.md]:
```suggestion
```

File: microsoft-365/compliance/inactive-mailboxes-in-office-365.md (hunk 4 above)
Comment (JohanFreelancer9) [inline microsoft-365/compliance/inactive-mailboxes-in-office-365.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review

**Comment by mypil** [comment]
@Tangela123 - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Pull request merged**